transforms/Makefile

```diff
@@ -78,4 +80,55 @@ workflow-upload::
 
 set-versions::
 	@# Help: Recursively make $@ in all subdirs
+	make set-pkg-version
 	@$(MAKE) RULE=$@ .recurse
+
+set-pkg-version:
```

Comment:
Maybe some help text for these new rules?

Reply:
Added